MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+# MANIFEST.in is defined so we can include non-Python (e.g. JS) files
+# in the built distribution.
+# See https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html
+graft src/django_components/static
```

Comment:
Since the JS-side manager code is compiled into `django_components.min.js`, which is NOT a python file, then  we need to use `MANIFEST.in` to configure what files are packaged into a distribution.

This adds all the static files. So that, when user installs `django_components`, and adds it to `INSTALLED_APPS`, then `django_components.min.js` will be included and found.

Here's how the result looks:
<img width="764" alt="Screenshot 2024-09-21 at 09 00 25" src="https://github.com/user-attachments/assets/4c10c9bd-1913-4a91-849e-2c982bf046af">